### ISSUE TITLE
Keyboard accelerators on MenuStrip are ignored

### ISSUE THREAD

**Issue by albahari** [dotnet/winforms#1021]
* .NET Core Version: Preview6 27720-09
* Have you experienced this same bug with .NET Framework?: No

**Problem description:**

Keyboard accelerators on MenuStrip are ignored. For instance, when the File menu is open and you press 'S' to save, nothing happens, despite the menu item text being "&Save". Also, the keyboard arrows don't work fully - when you press up-arrow on the first menu item, it doesn't cycle to the last item on the menu as it does in .NET Framework (and every other Windows app). This makes it hard work to navigate to the last item on the menu.

(Note that this bug was likely introduced around Preview5. In earlier .NET Core previews, I recall menu navigation working correctly.)

**Minimal repro:**

```
var file = new ToolStripMenuItem ("File");

file.DropDownItems.Add ("&New");
file.DropDownItems.Add ("&Open");
file.DropDownItems.Add ("&Save");
file.DropDownItems.Add ("Save &as");

var main = new MenuStrip ();
main.Items.Add (file);

var form = new Form { MainMenuStrip = main };
form.Controls.Add (main);

form.Show();
```

**Comment by merriemcgaw** [comment]
@M-Lipin can you see if something with ToolStrip accessibility work impacted this?

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team help us identify what build regressed this behavior to help us narrow down a cause?

**Comment by Amy-Li02** [comment]
@merriemcgaw 
Issue #1 -  When the File menu is open and press 'S' to save, the menu item still shown on Form.
Issue #2 -  When pressing up-arrow on the first menu item, it doesn't cycle to the last item on the menu.

For Issue #1, this is a regression, cannot repro in .Net Core 3.0 SDK 3.0.100-preview4-011216 release build.
For Issue #2, verified in the latest .Net Core SDK 3.0.100.12499 - preview7 from master branch & preview6 release build & preview5 release build & preview4 release build, the issue all can reproduce.

**Comment by merriemcgaw** [comment]
@Amy-Li02 is issue num 2 a regression from .NET Framework?

**Comment by merriemcgaw** [comment]
@M-Lipin Can you take a look at this for Preview 8

**Comment by Amy-Li02** [comment]
@merriemcgaw Issue #2 is a regression from .NET 4.8 03627.00 from NETFXRel1 branch, however it works well in .NET 4.7.2.

**Comment by merriemcgaw** [comment]
@Amy-Li02 are you saying that this regressed in 4.8?

**Comment by Amy-Li02** [comment]
@merriemcgaw Yes, Issue #2 regressed in 4.8.

**Comment by merriemcgaw** [comment]
@Amy-Li02 please file a 4.8 bug ASAP if one isn't filed already. This is one we'll absolutely need to service. I'll make sure @M-Lipin and @vladimir-krestov are looking at it actively. Please mark it as a11y-MAS

**Comment by Amy-Li02** [comment]
@merriemcgaw we have filed a 4.8 bug about Issue#2.
